Library/Formula/shairport-sync.rb

```diff
@@ -0,0 +1,58 @@
+class ShairportSync < Formula
+  desc "AirTunes emulator. Shairport Sync adds multi-room capability."
+  homepage "https://github.com/mikebrady/shairport-sync"
+  url "https://github.com/mikebrady/shairport-sync/archive/2.4.1.tar.gz"
+  sha256 "d2af0e9ee86f3c24f08ddb43ee73c79c5426b52a11d83a11f6c71ed2023df32d"
+
+  head "https://github.com/mikebrady/shairport-sync.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "openssl"
+  depends_on "popt"
+  depends_on "libsoxr"
+  depends_on "libao"
+  depends_on "libdaemon"
+  depends_on "libconfig"
+
+  # Fix configure.ac script for searching dns_sd system library
+  patch :DATA
```

Comment:
Now that it’s merged upstream you can use the direct patch URL:

```rb
patch do
    url "https://github.com/mikebrady/shairport-sync/commit/cfde9188910f48fdb31bdfc63a09e34137f8f6d9.patch"
    sha256 "bf9cc9c299bed01b93ae4bdb1525857af2fbe96144fbc5c5de266843dcaa39e4"
end
```

Thank you for making this PR upstream :clap: